core/domain/skill_domain.py

```diff
@@ -177,11 +181,19 @@ class SkillChange(change_domain.BaseChange):
     }]
 
 
+misconception_struct = TypedDict(
```

Comment:
Use class declaration of TypedDict, look into other files to see how it is done. Ditto elsewhere.